controller/users.go

```diff
@@ -418,7 +417,7 @@ func LoadKeyCloakIdentities(appl application.Application, users []account.User)
 		identity, err := loadKeyCloakIdentity(appl, user)
 		// if we can't find the Keycloak identity
 		if err != nil {
-			logrus.Error(err)
+			log.Error(nil, map[string]interface{}{"user": user, "err": err}, "unable to load user keycloak identity")
```

Comment:
Technically it should never happened. A user is supposed to have a KC identity anyway.